### PR TITLE
myauctions page and auctionshow page functional, and only use the moc…

### PR DESCRIPTION
**Pull request by hawk198723**
…kdata so far

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,3 +62,350 @@ app/assets/stylesheets/application.scss
 ```
 $ yarn add react-router-dom@5.3.0
 ```
+### Feb/19/2022
+> made a branch called 'index'
```
Comment (zan-rosenthal) [inline README.md:66]: I don't think you need to check these updates in to the README. The PR itself essentially serves as a log of work done. You could move your comments here into the description of the PR. Then, even once the PR is merged, you can see it in the closed PR's tab.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -4,35 +4,52 @@ import Home from './pages/Home'
...
+
+            <Route path="/my_auctions_route" render={(props) => <MyAuctions auctions={this.state.auctions} />} />
+            
+            <Route path="/auction_show_route/:id" render={(props) => {
+              let paramId = +props.match.params.id
+              let auction = this.state.auctions.find(auction => auction.id === paramId )
+              return <AuctionShow auction={auction}/>}} />
+
+            
+            <Route path="/auction_new_route" render={(props) => <AuctionNew createAuction={this.createAuction} />} />
```
Comment (zan-rosenthal) [inline app/javascript/components/App.js]: Is this the code that was throwing the error you posted here? https://sdlearn.slack.com/archives/C0325DHSY2V/p1645356049961859

You may need to set the `this` context for the handler. You can do that by using the `bind` method, i.e.:
`<AuctionNew createAuction={this.createAuction.bind(this)} />`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind

File: app/javascript/components/components/Header.js [hunk 3]
```diff
@@ -15,45 +15,36 @@ class Header extends Component {
...
+            {logged_in &&
+              <NavLink to= "/my_bids_route" className="nav-link">My Bids</NavLink>
+            }
+          </div>
+          <div className="nav-bar">
+              <Nav>
+                <NavItem>
+                  <NavLink to="/about" className="nav-link">About Us</NavLink>
+                </NavItem>
+              </Nav>
```
Comment (zan-rosenthal) [inline app/javascript/components/components/Header.js:47]: @hawk198723 I think you may have format on save enabled in VSCode. You, Kevin, and Raul should sync your VSCode settings so the diff on PR's in not as big. It looks like there were a lot of changes here, but looking at it, I think most of it was just indentation.

If you all install this VSCode extension: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode, then enable "Format on Save" in the VSCode settings, it should format all your code the same way.

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]

File: app/javascript/components/App.js (hunk 2 above)
Comment (sjproctor) [reply app/javascript/components/App.js]: Since they are using an arrow function you shouldn't need to bind.

**Review by sjproctor — APPROVED** [review]

**Pull request merged**